content/docs/developing-operators/tasks.md

```diff
@@ -47,6 +47,27 @@ tasks:
 
 **Note:** deleting non-existing resources is always successful. As of version 0.9.0, KUDO will not wait for the resource to be actually removed and will finish the task when the API server accepts the deletion request. So in case of Pods, Kubernetes imposes a default graceful termination period of 30 seconds, however, a delete-task will be done before that. KUDO 0.9.0 will not wait for [resource finalizers](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/) should they exist.
 
+
+## Toggle-Task
+
+Developing featrues for operators sometimes requires enabling and disabling resources.
+A toggle-task runs an apply-task or delete-task based on a parameter boolean value.
```

Comment:
thought: should we treat the fact that actually apply/delete tasks are used on the inside as an implementation detail? 🤔 
```suggestion
A toggle-task applies or deletes resources based on a parameter boolean value.
```